### PR TITLE
adding classification with BERT notebook

### PR DESCRIPTION
**Pull request by sanjanalreddy**
Libraries needed to run this lab

`pip install  "tf-models-official==2.8.0"` and `pip install "tensorflow-text==2.8.*"`

### CODE REVIEW

File: notebooks/text_models/solutions/classify_text_with_bert.ipynb [hunk 1]
```diff
@@ -0,0 +1,897 @@
+{
```
Comment (BenoitDherin) [inline notebooks/text_models/solutions/classify_text_with_bert.ipynb:1]: <p>* "to train a your BERT" -&gt; "to train a BERT"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (BenoitDherin) [inline notebooks/text_models/solutions/classify_text_with_bert.ipynb:1]: <ul><li>Alphabetic order in the second group maybe (official should go between matplotlib and tensorflow)</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (BenoitDherin) [inline notebooks/text_models/solutions/classify_text_with_bert.ipynb:1]: <p>* "hub.KerasLaye" --&gt; <code>hub.KerasLayer</code></p><p> <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (BenoitDherin) [inline notebooks/text_models/solutions/classify_text_with_bert.ipynb:1]: <p>* "It can be" --&gt; "it can be"</p><p>* Not sure I understand what this sentence means: "Since this text preprocessor is a TensorFlow model, It can be included in your model directly."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (BenoitDherin) [inline notebooks/text_models/solutions/classify_text_with_bert.ipynb:1]: <p>This is very clear!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (BenoitDherin) [inline notebooks/text_models/solutions/classify_text_with_bert.ipynb:1]: <ul><li>Something off with this sentence: "you can use just follow the model's url"</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (BenoitDherin) [inline notebooks/text_models/solutions/classify_text_with_bert.ipynb:1]: <p>Maybe add the dropout rate as an argument of <code>build_classifier_model</code> and increase it a little before training the model (the training curves I obtained are showing an overfit that could benefit from an increase dropout rate).  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (BenoitDherin) [inline notebooks/text_models/solutions/classify_text_with_bert.ipynb:1]: <ul><li>"evaluation metric" --&gt; "evaluation metrics" ?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (BenoitDherin) [inline notebooks/text_models/solutions/classify_text_with_bert.ipynb:1]: <p>Nice!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

**Review by BenoitDherin — APPROVED** [review]

File: notebooks/text_models/solutions/classify_text_with_bert.ipynb (hunk 1 above)
Comment (sanjanalreddy) [reply notebooks/text_models/solutions/classify_text_with_bert.ipynb:1]: <p>Done</p>

Comment (sanjanalreddy) [reply notebooks/text_models/solutions/classify_text_with_bert.ipynb:1]: <p>Re-phrased the sentence to make it clearer.</p>

Comment (sanjanalreddy) [reply notebooks/text_models/solutions/classify_text_with_bert.ipynb:1]: <p>Re-phrased</p>

Comment (sanjanalreddy) [reply notebooks/text_models/solutions/classify_text_with_bert.ipynb:1]: <p>Done</p>
